### DISCUSSION TITLE
Is there historical transit data available through Transitland?

### DISCUSSION THREAD

**Discussion by Mike-Joseph-Ball** [transitland/transitland#437] [Q&A] [answered]
I'm new to transitland and GTFS/GTFSRt

I want to know if there is historical data on bus times. Is it possible to find out the times a particular bus arrived at particular stops on a particular day? Or is there no authorities that save this kind of data? It seems like GTFSRt only really shows what is happening in real time and doesn't save any data. 

I am in the United States in Orange County, so I am mostly interested in OCTA and Metrolink. 

Thank you

**Comment by drewda** [comment, marked as answer]
Transitland APIs currently only utilizes the latest GTFS Realtime or GBFS feed.

Some agencies archive their own GTFS Realtime data. Interline also does so for some clients. For example: https://www.interline.io/blog/mtc-regional-stop-observations/ But the Transitland public platform only uses current/latest real-time data.

While it's not the question here, I'll also note for others' reference that Transitland does have a deep archive of _static_ GTFS feeds.
